api/v1alpha4/awsmachine_webhook.go

```diff
@@ -28,6 +28,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
+const (
```

Comment:
Feels like we need to be declaring these values in the API types. A bit like we do for something like [InstanceState](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/59a9bd2954d116e2af394e75d522634f7ebb8e3c/api/v1alpha4/types.go#L563:L609). What do you think?